iOS Client/ArticlesController/MBArticlesViewController.swift

```diff
@@ -109,73 +103,84 @@ class MBArticlesViewController: UIViewController, UITableViewDelegate, UITableVi
         case .error:
             print("Error: Loading articles")
         case .loaded(let data):
-            if data.count != articles.count {
```

Comment:
took out this condition since the count doesn't necessarily tell us whether or not the data is new.

Reply:
Yeah this is actually a pretty big topic of discussion on how to tell if you have new data. There are whole libraries to do table diffing and such. We can probably just get by with reloading everytime and if it seems to impact performance then we address it